alfresco-transform-imagemagick/alfresco-transform-imagemagick/src/main/java/org/alfresco/transformer/executors/ImageMagickCommandExecutor.java

```diff
@@ -76,6 +80,16 @@ protected RuntimeExec createTransformCommand()
         processProperties.put("MAGICK_HOME", ROOT);
         processProperties.put("DYLD_FALLBACK_LIBRARY_PATH", DYN);
         processProperties.put("LD_LIBRARY_PATH", DYN);
+
+        //Optional properties (see also https://imagemagick.org/script/resources.php#environment)
+        if (CODERS != null && !CODERS.isBlank())
+        {
+            processProperties.put("MAGICK_CODER_MODULE_PATH", CODERS);
+        }
+        if (CONFIG != null && !CONFIG.isBlank())
+        {
+            processProperties.put("MAGICK_CONFIGURE_PATH", CONFIG);
+        }
```

Comment:
Do both these have to be set in order to work? If they each work on their own then don't worry.
Do we want some error handling if this is the case?  
if (CODERS ==null AND CONFIG!=null){throw error("you need to supply both CODERS and CONFIG)}

Reply:
Please refer to this [comment ](https://github.com/Alfresco/alfresco-transform-core/pull/229#pullrequestreview-396635253) where we discussed the above with Jan